src/status_im/contexts/wallet/send/events.cljs

```diff
@@ -168,6 +169,7 @@
                                   to-address
                                   amount-in
                                   token-id
+                                  to-token-id
```

Comment:
should there not be a corresponding Status-Go update with this?

Reply:
There is, it's alteady merged. I mentioned it in the description.

Reply:
Yes but we should bump the version of status go in this pr too, no?